docs/setup.md

```diff
@@ -54,8 +54,6 @@ Get your OpenAI API key from: [https://platform.openai.com/account/api-keys](htt
             environment:
               MEMORY_BACKEND: ${MEMORY_BACKEND:-redis}
               REDIS_HOST: ${REDIS_HOST:-redis}
-            volumes:
-              - ./:/app
```

Comment:
Needs volume mounts for `auto_gpt_workspace`, `logs`, `auto-gpt.json`, `ai_settings.yaml`, `azure.yaml`, `CURRENT_BULLETIN.md`, some of them commented out so people can activate them when they need them.

Reply:
mh, mounting CURRENT_BULLETIN.md doesn't make sense, makes more sense to bake that into the Dockerfile

Reply:
> makes more sense to bake that into the Dockerfile

Absolutely not, that misses the whole point of the file.

Reply:
ok? because nobody is gonna update it in the runtime folder? 
if its the bulletin with the release -> dockerfile
if its pulled during startup -> why mount?

why is azure.yml essential, what am I missing?

i'm gonna skip auto-gpt.json, as that should be covered by using redis?

Reply:
> if its pulled during startup -> why mount?

Because the instructions say to run with `--rm`, meaning it won't be persisted in the container, and if it isn't mounted and persisted that way, the bulletin will show up as "new" on every run.

> why is azure.yml essential, what am I missing?

It is essential to users who use Azure.

> i'm gonna skip auto-gpt.json, as that should be covered by using redis?

yup